pkg/sentry/platform/ring0/defs_amd64.go

```diff
@@ -97,11 +137,7 @@ type CPUArchState struct {
 	// exception.
 	errorType uintptr
 
-	// gdt is the CPU's descriptor table.
-	gdt descriptorTable
-
-	// tss is the CPU's task state.
-	tss TaskState64
+	*kernelEntry
```

Comment:
Does this need to be embedded?

Reply:
there are too much c.tss c.gdt in the code, embedding it to keep the code unchanged.